src/Configuration.ts

```diff
@@ -46,7 +46,7 @@ export const CONFIG = {
       IDENTIFICATIVO_PSP: "AGID_01",
       IDENTIFICATIVO_INTERMEDIARIO_PSP: "97735020584",
       IDENTIFICATIVO_CANALE: "97735020584_02",
-      PASSWORD: process.env.PAGOPA_PASSWORD || "nopassword"
+      PASSWORD: process.env.PAGOPA_PASSWORD
```

Comment:
is this making the server to fail to start or is this just not sending a password in the soap requests?